originals/u.yaml

```diff
@@ -1,15 +1,3 @@
-- name: 'UFO: Enemy Unknown'
```

Comment:
this can be an alternate name for XCOM UFO Defense (add under the `names` list)

Reply:
This is already [added](https://github.com/opengaming/osgameclones/blob/19297f299255ca4de4f34184925407418d82fdb6/originals/x.yaml#L27)